cards/assessments/iama.yaml

```diff
@@ -0,0 +1,381 @@
+name: IAMA
+date: "2025-03-25"
+contents:
+  - question: "Licht uw voorstel voor het gebruik/de inzet van een algoritme toe. Wat is de aanleiding hiervoor geweest? Voor welk probleem moet het algoritme een oplossing bieden?"
+    answer: "Lorem ipsum dolor sit amet, officia excepteur ex fugiat reprehenderit enim labore culpa sint ad nisi Lorem pariatur mollit ex esse exercitation amet. Nisi anim cupidatat excepteur officia. Reprehenderit nostrud nostrud ipsum Lorem est aliquip amet voluptate voluptate dolor minim nulla est proident. Nostrud officia pariatur ut officia. Sit irure elit esse ea nulla sunt ex occaecat reprehenderit commodo officia dolor Lorem duis laboris cupidatat officia voluptate. Culpa proident adipisicing id nulla nisi laboris ex in Lorem sunt duis officia eiusmod. Aliqua reprehenderit commodo ex non excepteur duis sunt velit enim. Voluptate laboris sint cupidatat ullamco ut ea consectetur et est culpa et culpa duis."
+    remarks: Remark
+    authors:
+      - name: "John Doe"
+      - name: "Jane Doe"
+    timestamp: "1711630721"
```

Comment:
So are iso8601 datetimes in UTC, they look like this: 2024-04-16T12:44:46Z

Reply:
Yes exactly. So then `16T12:44:46+00:00` as in the follow-up remark should resolve this, with the exception that `isoformat()` emits a `+00:00` instead of a `Z`. But this can be adjusted to be consistent with your example (and ISO8601).

Reply:
You're right, `isoformat()` sadly emits `+00:00` instead of `Z` (even though this will always refer to the same date-time, it semantically not the same thing). 

In the past I've used `.isoformat().replace("+00:00", "Z")`